.travis.yml

```diff
@@ -3,4 +3,3 @@ rvm:
 - 2.1
 - 1.9.3
 - jruby
-- jruby-head
```

Comment:
It depends what version of RVM is being used in Travis CI. `jruby-head` in older versions refers to JRuby 9k, which has since been released and is the current version of JRuby.

In newer versions of RVM, `jruby` refers to `jruby-9.0.0.0`. We should definitely still be testing on JRuby 1.7.x (1.7.21) and JRuby 9k